### ISSUE TITLE
please stop URL rewriting when accessing the page

### ISSUE THREAD

**Issue by nakic** [tyrasd/overpass-turbo#563]
As it stands, a user can prepare a query, click share and get a direct link to that query. When the link is opened, the URL is rewritten from something like https://overpass-turbo.eu/s/1gwb to https://overpass-turbo.eu/.

This is by far the single most problematic part of the overpass turbo UI. Looking up previous queries in the browser URL field is really hard, as is bookmarking. To bookmark a link (using Firefox, at least), a user basically has to enter a query such as https://overpass-turbo.eu/s/1gwb, bookmark the page, open bookmark management, find the link and update the URL with the full query URL. That is really cumbersome.

Can the link stay https://overpass-turbo.eu/s/1gwb without being rewritten until e.g. pressing "run" or "share"?

**Comment by mmd-osm** [comment]
You can right-click on a link and hit "Bookmark link". This also works in overpass turbo when you hit the "Share" button, and then right click on the blue "Link" in the screenshot blow.

![csdv3](https://user-images.githubusercontent.com/5842757/156415525-028ea050-bf05-4ea6-b2ee-acd81a8037ee.png)

**Comment by nakic** [comment]
Hi mmd, thanks for the tip: I didn't know. Still, the bookmark issue is just one in a crowd. I (and I assume others) use the address bar as a quick search of history. If I type "over dk2" I would like the browser to find http://overpass-turbo.eu/s/dk2 , but if I never see the full query URL in the address bar, I can't really notice the URL ends in dk2.

I use a browser extension to suspend inactive browser tabs after a while, which works well with most sites, but doesn't work with overpass at all because if I have multiple queries open in a series of tabs, once the tabs are suspended, when I access them again, they will all show https://overpass-turbo.eu/ instead of the specific query that was loaded before.

Basically, I don't see the value of rewriting the URL to http://overpass-turbo.eu/ in the first place and there are quite a few usability downsides. Am I missing something?
